easybuild/easyblocks/i/impi.py

```diff
@@ -126,9 +126,12 @@ def install_step(self):
             run_cmd(cmd, log_all=True, simple=True)
 
         # recompile libfabric (if requested)
-        if impiver >= LooseVersion('2019') and self.cfg['libfabric_rebuild']:
+        # some Intel MPI versions (like 2019 update 6) no longer ship libfabric sources
+        libfabric_path = os.path.join(self.installdir, 'libfabric')
+        if (impiver >= LooseVersion('2019') and self.cfg['libfabric_rebuild'] and
+                os.path.exists(os.path.join(libfabric_path, 'src.tgz'))):
```

Comment:
A couple of things I don't like here:

* wrapping the if condition (especially while there's another one below)
* hardcoding the `src.tgz` in two places
* no logging why we're skipping `libfabric` rebuild even though it's requested

fixed in https://github.com/ComputeCanada/easybuild-easyblocks/pull/5